lib/root_controller.js

```diff
@@ -0,0 +1,174 @@
+var Tessel = require('./tessel/tessel');
+var logs = require('./logs');
+var debug = require('debug')('controller');
+
+// create an object for t2 root steps for beeing more handy about unit testing of the T2 root command itself.
+var controller = {};
+controller.ssh = {
```

Comment:
@Student007 You have a good point about making it modular, but I think we shouldn't have just one function be modular. We should either modularize the whole file or none of the file. `controller.js` is still pretty small so I think we're okay keeping it in that file for now :)